packages/react/src/accordion/index.tsx

```diff
@@ -3,12 +3,28 @@ import { CdsAccordionContent as AccordionContent } from '@cds/core/accordion';
 import { CdsAccordionHeader as AccordionHeader } from '@cds/core/accordion';
 import { CdsAccordionPanel as AccordionPanel } from '@cds/core/accordion';
 import '@cds/core/accordion/register';
-import { createComponent } from '../converter/react-wrapper.js';
+import { createComponent } from '@lit-labs/react';
+import * as React from 'react';
 
-export const CdsAccordion = createComponent('cds-accordion', Accordion);
-export const CdsAccordionPanel = createComponent('cds-accordion-panel', AccordionPanel, {
+// add react util here in all components.
+// Document that it needs to be part of each component in React.
+
+export const CdsAccordion = createComponent(React, 'cds-accordion', Accordion);
+export const CdsAccordionPanel = createComponent(React, 'cds-accordion-panel', AccordionPanel, {
   onExpandedChange: 'expandedChange',
   onCdsMotionChange: 'cdsMotionChange',
 });
-export const CdsAccordionHeader = createComponent('cds-accordion-header', AccordionHeader);
-export const CdsAccordionContent = createComponent('cds-accordion-content', AccordionContent);
+export const CdsAccordionHeader = createComponent(React, 'cds-accordion-header', AccordionHeader);
+export const CdsAccordionContent = createComponent(React, 'cds-accordion-content', AccordionContent);
+
+/**
+ * Setting displayName on the prototype is a workaround for now.
+ * See: https://github.com/lit/lit/issues/2154
+ * Follow: https://github.com/lit/lit/pull/2155
+ */
+CdsAccordion.displayName = 'CdsAccordion';
```

Comment:
Until the wrapper tools are modified to add this, I'm doing it this way to still give React dev tools more information about these components.

Reply:
We used to handle this in our fork of the createComponent fn. Basically, with this line, they see <CdsButton> in the react dev tools instead of <ForwardRef> (I think, i forget exactly what surfaces without it, it might be undefined). It's not a blocker if we want to remove it, there is already an issue / PR for it in the lit repo.

Reply:
Without this the react dev tools report the component as the base class name in the react dev tools rather than the name of the component